### PR TITLE
add basic JavaScript test for BlacklightRangeLimit

### PR DESCRIPTION
**Pull request by mejackreed**
Hoping just having a basic test here will be useful prevent regressions

### CODE REVIEW

**Comment by jrochkind** [comment]
This seems like a great idea, not sure if it was ever green?  @mejackreed , thoughts on this? Worth trying to rebase to green?  Just close it?

I'd love to get the list of PR's empty (merged or closed), then maybe do a 7.1 release?

**Comment by jrochkind** [comment]
Merged master in, let's see if we can get travis green...

**Comment by jrochkind** [comment]
I merged master in, hoping it will result in travis passing. 

Ah, this is still using chromedriver-helper though, which is EOL. 

@mejackreed are you interested in working on this at the moment? I'm not sure I have time to take it all the way at the moment, and am not sure it's needed for a 7.1 release (although it'd be nice at any time). 

I'm just trying to go through old PR's and get things taken care of, in preparation hopefully for a 7.1 release.

**Comment by mejackreed** [comment]
@jrochkind updated to remove the deprecated package

**Comment by jrochkind** [comment]
Thanks @mejackreed !

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -5,6 +5,20 @@
...
+    chromeOptions: {
+      args: %w[headless disable-gpu no-sandbox window-size=1280,1696]
+    }
+  )
+  Capybara::Selenium::Driver.new(app,
+                                 browser: :chrome,
+                                 desired_capabilities: capabilities)
+end
+
+Capybara.javascript_driver = :headless_chrome
```
Comment (jrochkind) [inline spec/spec_helper.rb]: I believe with recent versions of capybara, there is no need to do this custom setup, you just do `Capybara.javascript_driver = :selenium_chrome_headless`, it comes out of the box as a registered driver under that name. 

Might be safer and more future-proof to use the default settings that Capybara suggests with that pre-registered driver, rather then trying to set it up ourselves?

I may go ahead and try it?

**Pull request merged**